docs/components/box/test.html

```diff
@@ -0,0 +1,31 @@
+---
+title: Testing - Boxes
+---
+{% extends 'test.njk' %}
+{% block content %}
+<section>
+  <nav class="hxBreadcrumb">
+    <a href="components/box">Box</a>
+    <hx-icon class="delimiter" type="angle-right"></hx-icon>
+    <a href="#">{{page.title}}</a>
+  </nav>
+</section>
+
+<section>
+  <header>
+    <h2>Status inside Div</h2>
+  </header>
+
+  <table class="hxTable">
```

Comment:
Probably don't need the table 😄. Just do...

```html
<hx-div>
  <hx-status>hx-div &gt; hx-status</hx-status>
</hx-div>
```

Reply:
test updated 😄